avocado_virt/plugins/virt_bootstrap.py

```diff
@@ -82,9 +83,10 @@ def run(self, args):
                           'it (205 MB). Please wait...', jeos_dst_path)
```

Comment:
We either need to change or none.  I'd change none at this point.

Reply:
I'd go with none or hundreds (to give some estimate)

Reply:
I meant "need to change **all** or none".  Anyway, up to you.